README: remove list of clients (#7381)
tldr-pages/tldr: README.md

out the [PDF version](https://tldr.sh/assets/tldr-book.pdf).

> There are also various other clients provided by the community,

Reviewer: Since we replaced the whole list by just two sentences, they can quickly be overlooked. Maybe we should put it in a separate section, or at least highlight it a bit:
```suggestion
There are also **various other clients** provided by the community,
```

Author: Sure i can do it

both for the command-line and for other platforms.

> There is a comprehensive
> [list of clients in our Wiki](https://github.com/tldr-pages/tldr/wiki/tldr-pages-clients).

Reviewer: Starting both sentences with the same phrase doesn't sound good. How about
```suggestion
For a comprehensive list of clients, head over to our [Wiki](https://github.com/tldr-pages/tldr/wiki/tldr-pages-clients).
```
